ci/check_enterprise_merge.sh

```diff
@@ -65,6 +65,14 @@ fi
 # undo partial merge
 git merge --abort
 
+# If we have a conflict on enterprise merge on the master branch, we have a problem.
+# Provide an error message to indicate that enterprise merge is needed.
+if [[ $PR_BRANCH = master ]]; then
```

Comment:
At first I thought we should write "master" but it seems like the current version also works(I tested with bash)

Reply:
This is the beauty of `[[ ]]` operators